<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUsercfgDlg</class>
 <widget class="QDialog" name="QUsercfgDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>80</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>角色类别描述：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cbxRoletype">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_rdbtnGrplder">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>是否组长</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>角色类别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>描述：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editCode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editRoledesc">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_editDesc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btOk">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btCancel">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>说明：密码必须包含大写字母、小写字母、数字，且不得少于8位。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_btCancel</sender>
   <signal>clicked()</signal>
   <receiver>QUsercfgDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btOk</sender>
   <signal>clicked()</signal>
   <receiver>QUsercfgDlg</receiver>
   <slot>onBtOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtOK()</slot>
 </slots>
</ui>
